ethereumetl/streaming/item_exporter_creator.py

```diff
@@ -23,29 +23,30 @@
 from blockchainetl.jobs.exporters.console_item_exporter import ConsoleItemExporter
 from blockchainetl.jobs.exporters.multi_item_exporter import MultiItemExporter
 
-DEFAULT_KAFKA_TOPIC_MAPPING = {
-    'block': 'ethereum_blocks',
-    'transaction': 'ethereum_transactions',
-    'log': 'ethereum_logs',
-    'token_transfer': 'ethereum_token_transfers',
-    # todo(shashank): migrate token_transfers_v2 to new topic
-    'token_transfer_v2': 'token_transfers_v2',
-    'trace': 'ethereum_traces',
-    'contract': 'ethereum_contracts',
-    'token': 'ethereum_tokens',
+CHAIN_ID_TOPIC_PREFIX_MAPPING = {
+    1: 'ethereum',
+    4: 'ethereum_rinkeby',
```

Comment:
I think we can just stop using Rinkeby

Reply:
yeah once this PR is merged, I will change the provider url